src/session_manager.rs

```diff
@@ -863,18 +847,6 @@ async fn set_online(client_id: i32, value: bool) -> Result<(), types::Error> {
     .await
 }
 
-/// Helper function to enable/disable animated emoji for a client.
-async fn disable_animated_emoji(client_id: i32, value: bool) -> Result<(), types::Error> {
```

Comment:
That option gets cached, so I can set it to opposite value if needed. Otherwise You can just delete db folder

Reply:
I suggest keeping it for now and set it to false. Many people tried Telegrand already and they will think that animated emojis don't work even once this is merged.